scss/blockQuote.module.scss

```diff
@@ -0,0 +1,11 @@
+.blockQuote{
+    background: #f9f9fa;
+     padding: 0.5em 1em
+}
+.blockQuote>p{
+    margin-top: 1em;
+}
+.blockQuote__bulb{
+    align-self: center;
+    height:1.5rem;
+}
```

Comment:
Please fix the SCSS files formatting.

Reply:
Fixed.
But I wonder what do you guys use for SCSS formatting? There is no `.prettierignore` in repo, so it's not `yarn prettier --write .` And vscode didn't recognize scss files unless I explicitly added `parser: "scss"` into prettier options.

Reply:
Oh you're right, prettier parses the `.scss` files as typescript, we need an override on our prettier config.

Also to format the individual file I'm working on I usually do it through the Prettier VS Code extension and with the Format Document hotkey (Alt + Shift + F).

Reply:
Fixed that on #586